index.ts

```diff
@@ -37,34 +55,18 @@ const foo = async () => {
     distributableRewards: cycleData.cycleRewards,
   });
 
-  const provider = createProvider();
   const { delegatorPayments, feeIncomePayments, bondRewardPayments } =
-    result.cycleReport;
-
-  const transactions = _.concat(
```

Comment:
Just to clarify – this was a temporary piece of code to allow payments on testnet while this solution is being developed. I am using data from real mainnet bakers to test and the payments of course fail when they can't find a smart contract address on testnet.